### ISSUE TITLE
drag: wrong window is dragged

### ISSUE THREAD

**Issue by pawamoy** [qtile/qtile#1413]
See this screen recording:

![r](https://user-images.githubusercontent.com/3999221/63952715-57d96380-ca80-11e9-9bc9-abed139e8476.gif)

If the window I try to drag is on both screens, then it drags the one that is *only* on this screen. When it is entirely contained within the screen, it works correctly.

**Comment by ramnes** [comment]
Yes, I've already encountered this. Thanks for the great bug report!

**Comment by ervinpopescu** [comment]
TODO: Need to test #4557 after rebase. Don't mind me, just updating this to get it down the stack in the updated-asc sort :)

**Comment by lolorc** [comment]
also reproduced here on 4 screens setup only when cursor_warp=True, #4557 makes it worse

**Comment by tych0** [comment]
Hmm, can you guys share some config snippets? I guess these are `Drag` keybinds? The relevant code is here:

https://github.com/qtile/qtile/blob/2e57a4a9ebb9600109d41cc185b7ec720a3de691/libqtile/core/manager.py#L864-L884

...it's a little crazy that it does an IPC call if you supply `start`, but let's ignore that. It looks like `_focus_hovered_window()` eventually calls this:

https://github.com/qtile/qtile/blob/2e57a4a9ebb9600109d41cc185b7ec720a3de691/libqtile/backend/x11/core.py#L924-L927

which doesn't seem all that wrong, but maybe there's some nuance here?

**Comment by tych0** [comment]
Does https://github.com/qtile/qtile/commit/d798efaca758cd6e76b658386597e89e60b768ea fix it?
